src/System.Management.Automation/cimSupport/cmdletization/ScriptWriter.cs

```diff
@@ -1353,7 +1356,10 @@ private static MethodParameterBindings GetMethodParameterKind(CommonMethodMetada
             bool firstParameterSet = true;
             foreach (string parameterSetName in MultiplyParameterSets(GetMethodParameterSet(method), InstanceMethodParameterSetTemplate, commonParameterSets, queryParameterSets))
             {
-                if (!firstParameterSet) output.Write(", ");
+                if (!firstParameterSet)
+                {
+                    output.Write(", ");
+                }
                 firstParameterSet = false;
```

Comment:
Our code guidance is to have blank line after `}`.
I'd prefer we turn on Roslyn analyzer rule for this in follow PR.

Reply:
fixed